packages/shared/src/components/CloseButton.tsx

```diff
@@ -1,15 +1,25 @@
 import classNames from 'classnames';
 import React, { forwardRef, ReactElement, Ref } from 'react';
-import { Button, StyledButtonProps, ButtonProps } from './buttons/Button';
+import {
+  Button,
+  StyledButtonProps,
+  ButtonProps,
+  ButtonSize,
+} from './buttons/Button';
 import CloseIcon from './icons/Close';
 
+type CloseButtonProps = StyledButtonProps &
+  ButtonProps<'button'> & {
+    size?: ButtonSize;
```

Comment:
Should we not keep using the `buttonSize` as it is already available from the `ButtonProps`

Reply:
Great catch. Fixed 👍